# node
controller_rate: 240.0

# subscriber topics
actuator_commands_topic: /bicopter/actuator_commands
imu_readings_topic: /bicopter/imu_readings

# controller
gains:
  roll:
    kp: 300.0
    kd: 30.0
  pitch:
    kp: 700.0
    kd: 100.0
  yaw:
    kp: 100.0  # no yaw position control
    kd: 10.0



